stl.go

```diff
@@ -351,18 +363,18 @@ func parseGSIBlock(b []byte) (g *gsiBlock, err error) {
 	}
 
 	// Creation date
-	if v := strings.TrimSpace(string(b[224:230])); len(v) > 0 {
+	if v := strings.TrimSpace(string(b[224:230])); len(v) > 0 {		
 		if g.creationDate, err = time.Parse("060102", v); err != nil {
-			err = errors.Wrapf(err, "astisub: parsing date %s failed", v)
-			return
+			err = errors.Wrapf(err, "astisub: parsing date %s failed", v)			
+			g.creationDate=time.Now()
```

Comment:
I don't like seeing this error ignored. Can you explain why you're ignoring this error by describing your use case ?

Reply:
We have old stl files that they do not have this field filled. It is not critical information for us